### PR TITLE
Updated the Tutorial

### PR DESCRIPTION
**Pull request by AliMahmoudzadeh**
# PR into Azure/azureml-examples

## Checklist

I have:

- [ x] read and followed the contributing guidelines

## Changes
- Added various links to the AML documentation

fixes #
- Updated the dependencies in the environment

### CODE REVIEW

**Review by jfomhover — APPROVED** [review]

File: tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb [hunk 1]
```diff
@@ -206,13 +206,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This code just created a `Data` asset, it is ready to be consumed as an input by the pipeline that you'll define in the next sections. In addition, you can register the dataset to your workspace so it becomes reusable across pipelines.\n",
+    "This code just created a `Data` asset, it is ready to be consumed as an input by the pipeline that you'll define in the next sections. In addition, you can register the data to your workspace so it becomes reusable across pipelines.\n",
     "\n",
-    "Registering the dataset will enable you to:\n",
+    "Registering the Data asset will enable you to:\n",
```
Comment (sdgilley) [inline tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb:211]:
```suggestion
    "Registering the data asset will enable you to:\n",
```

File: tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb [hunk 2]
```diff
@@ -206,13 +206,13 @@
...
     "\n",
-    "* Reuse and share the dataset in future pipelines\n",
-    "* Use versions to track the modification to the dataset\n",
-    "* Use the dataset from Azure ML designer, which is Azure ML's GUI for pipeline authoring\n",
+    "* reuse and share the Data asset in future pipelines\n",
```
Comment (sdgilley) [inline tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb:213]:
```suggestion
    "* reuse and share the data asset in future pipelines\n",
```

File: tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb [hunk 3]
```diff
@@ -206,13 +206,13 @@
...
+    "* use versions to track the modification to the Data asset\n",
```
Comment (sdgilley) [inline tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb:214]:
```suggestion
    "* use versions to track the modification to the data asset\n",
```

File: tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb [hunk 4]
```diff
@@ -1018,9 +1024,9 @@
    "source": [
     "## Deploy the model as an online endpoint\n",
     "\n",
-    "Now deploy your machine learning model as a web service in the Azure cloud.\n",
+    "Now deploy your machine learning model as a web service in the Azure cloud, an [`online endpoint`](https://docs.microsoft.com/en-us/azure/machine-learning/concept-endpoints).\n",
```
Comment (sdgilley) [inline tutorials/e2e-ds-experience/e2e-ml-workflow.ipynb:1027]:
```suggestion
    "Now deploy your machine learning model as a web service in the Azure cloud, an [`online endpoint`](https://docs.microsoft.com/azure/machine-learning/concept-endpoints).\n",
```

**Review by sdgilley — CHANGES_REQUESTED** [review]

**Comment by AliMahmoudzadeh** [comment]
@sdgilley, I committed your suggestions, but it seems that the merge is still blocked, would you please manually approve if it is acceptable now. Thanks.

**Review by sdgilley — APPROVED** [review]

**Pull request merged**